lib/panoptes-subject-uploader.iced

```diff
@@ -30,13 +30,20 @@ findImagesFiles = (searchDir, metadata) ->
   imageFiles
 
 findImagesURLs = (metadata) ->
-  imageURLs = []
+  httpsImageURLs = []
   for key, value of metadata
+
     httpsRegexPattern = /^(https):\/\//i
     fileRegexPattern = /([^\/]+.(?:jpg|jpeg|gif|png|svg|mp4|txt))$/i
-    if httpsRegexPattern.test(value) and fileRegexPattern.test(value)
-      imageURLs.push value
-  imageURLs
+
+    if fileRegexPattern.test(value)
```

Comment:
We need handling if this returns false. Subjects are still created if there is a valid https URL found. For instance, I tried using `https://i.imgur.com/` in my test csv and it created and linked the subject.